pkg/operator/manualapprovalgate.go

```diff
@@ -44,3 +57,99 @@ func EnsureManualApprovalGateExists(clients mag.ManualApprovalGateInterface, nam
 	})
 	return ks, err
 }
+
+func StartApprovalGatePipeline() {
+	cmd.MustSucceed("tkn", "pipeline", "start", "manual-approval-pipeline")
+	log.Println("Waiting 10sec to start the pipeline")
+	cmd.MustSuccedIncreasedTimeout(time.Second*130, "sleep", "10")
+}
```

Comment:
We usually define pipeline run in a yaml fine and then use `Apply <table>` or `Create <table>` step. 
Even if you need to do it this way, there is a function `StartPipeline()` in `pkg/tkn/tkn.go`

Reply:
Done